src/main/java/org/folio/circulation/resources/DueDateNotRealTimeScheduledNoticeProcessingResource.java

```diff
@@ -0,0 +1,84 @@
+package org.folio.circulation.resources;
+
+import static java.lang.Math.max;
+import static org.folio.circulation.support.ResultBinding.mapResult;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.folio.circulation.domain.MultipleRecords;
+import org.folio.circulation.domain.notice.schedule.DueDateNotRealTimeScheduledNoticeHandler;
+import org.folio.circulation.domain.notice.schedule.ScheduledNotice;
+import org.folio.circulation.domain.notice.schedule.ScheduledNoticeGroupDefinition;
+import org.folio.circulation.domain.notice.schedule.ScheduledNoticesRepository;
+import org.folio.circulation.domain.notice.schedule.TriggeringEvent;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CqlSortClause;
+import org.folio.circulation.support.CqlSortBy;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+
+import io.vertx.core.http.HttpClient;
+
+public class DueDateNotRealTimeScheduledNoticeProcessingResource extends ScheduledNoticeProcessingResource {
+
+  public DueDateNotRealTimeScheduledNoticeProcessingResource(HttpClient client) {
+    super("/circulation/due-date-not-real-time-scheduled-notices-processing", client);
+  }
+
+  @Override
+  protected CompletableFuture<Result<MultipleRecords<ScheduledNotice>>> findNoticesToSend(
+    ScheduledNoticesRepository scheduledNoticesRepository, int limit) {
+
+    List<CqlSortClause> cqlSortClauses = Stream.of(
```

Comment:
@dmytrotur, can we declare the list as a class field or constant?

Reply:
Moved to constant